platform/vs/docker-sonic-vs.mk

```diff
@@ -16,10 +16,7 @@ $(DOCKER_SONIC_VS)_DEPENDS += $(SWSS) \
                               $(SONIC_HOST_SERVICES_DATA) \
                               $(IPROUTE2)
 
-# swsssdk is a dependency of sonic-py-common
-# TODO: sonic-py-common should depend on swsscommon instead
-$(DOCKER_SONIC_VS)_PYTHON_WHEELS += $(SWSSSDK_PY3) \
-                                    $(SONIC_PY_COMMON_PY3) \
+$(DOCKER_SONIC_VS)_PYTHON_WHEELS += $(SONIC_PY_COMMON_PY3) \
```

Comment:
>SONIC_PY_COMMON_PY3 [](http://example.com/codeflow?start=38&length=19)

Do you need to change the file rules/sonic-py-common.mk?

Reply:
No, sonic-py-common need following line to run UT with swsssdk, it's not a runtime dependency:

$(SONIC_PY_COMMON_PY3)_DEPENDS += $(SWSSSDK_PY3)